### PR TITLE
[Fix] fix missing deploy_core

### PR DESCRIPTION
**Pull request by AllentDan**
#79

### CODE REVIEW

**Comment by lvhan028** [comment]
I think it's better for us to dig more about `rpath` and `runpath`. 
From https://www.qt.io/blog/2011/10/28/rpath-and-runpath, it seems '-Wl,--disable-new-dtags' is not a recommended method.

**Comment by AllentDan** [comment]
> I think it's better for us to dig more about `rpath` and `runpath`. From https://www.qt.io/blog/2011/10/28/rpath-and-runpath, it seems '-Wl,--disable-new-dtags' is not a recommended method.

Maybe we can remind users to add our lib to LD_LIBRARY_PATH

**Comment by lvhan028** [comment]
> > I think it's better for us to dig more about `rpath` and `runpath`. From https://www.qt.io/blog/2011/10/28/rpath-and-runpath, it seems '-Wl,--disable-new-dtags' is not a recommended method.
> 
> Maybe we can remind users to add our lib to LD_LIBRARY_PATH

Agreed

File: demo/csrc/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,11 @@ project(mmdeploy-example)
 find_package(OpenCV REQUIRED)
 find_package(MMDeploy REQUIRED)
 
+set(MMDeploy_LIBS ${MMDEPLOY_LIBS}
```
Comment (lvhan028) [inline demo/csrc/CMakeLists.txt:8]: How about ```target_link_libraries(${name} ${MMDeploy_LIBS} -Wl,--disable-new-dtags ...)```?

**Review by lvhan028 — APPROVED** [review]
LGTM

**Review by lzhangzz — APPROVED** [review]

**Pull request merged**